pycloudlib/oci/instance.py

```diff
@@ -83,22 +83,18 @@ def delete(self, wait=True):
         if wait:
             self.wait_for_delete()
 
-    def restart(self, wait=True, **kwargs):
-        """Restart the instance.
-
-        Args:
-            wait: wait for the instance to be fully started
-        """
-        self.compute_client.instance_action(self.instance_data.id, 'RESET')
-        if wait:
-            self.wait()
-
     def shutdown(self, wait=True, **kwargs):
         """Shutdown the instance.
 
         Args:
             wait: wait for the instance to shutdown
         """
+        # Because it's like 1995 or something...
+        # But seriously, without a sync (or 5), if you power down too quickly
+        # after first boot, all the keys created in /etc/ssh will turn into
+        # zero-byte files and you'll be permanently kicked out of your
+        # instance with your only hope being serial console access.
+        self.execute('sync; sync; sync; sync; sync')
```

Comment:
LOL - is the number of syncs here specific? As in: would 4 work? or 3? Or do we need all 5 to ensure this will work?